src/js/utils/specs/ajax-helpers-spec.js

```diff
@@ -0,0 +1,166 @@
+define(
+    [
+        'jquery',
+        'sinon',
+        '../../utils/spec-helpers/ajax-helpers.js'
+    ],
+    function($, sinon, AjaxHelpers) {
+        'use strict';
+
+        describe('AjaxHelpers', function() {
+            var testUrl = 'https://example.com',
+                testData = {foo: 'bar', fizz: 'buzz'},
+                testQuerystring = $.param(testData);
+
+            describe('fakeServer', function() {
```

Comment:
Ah, ok!  I don't have strong opinions, but find personally, TODOs sometimes sit around for a while and I just end up feeling guilty about it.  Maybe delete and add a story for adding the tests in?